### PR TITLE
ENCD-5341 update deployment tags

### PR DESCRIPTION
**Pull request by caseylitton**

### CODE REVIEW

**Review by keenangraham — DISMISSED** [review]

**Comment by keenangraham** [comment]
Could also have an auto_resize with specified instance?

**Comment by caseylitton** [comment]
> Could also have an auto_resize with specified instance?

Added auto_resize = c5.4xlarge .  Will = 'na' when we don't want to resize.

File: src/encoded/commands/deploy.py [hunk 1]
```diff
@@ -98,34 +98,98 @@
...
+        profile_name='default',
+        dry_run=False,
+        arm_arch=False,
+        cluster_master=False,
+    ):
+    # Defaults to demo development
+    tags_dict = {
+        # Org
+        'project': 'encoded',
+        'section': 'app',
```
Comment (hitz) [inline src/encoded/commands/deploy.py:123]: what is "section" supposed to represent?

File: src/encoded/commands/deploy.py [hunk 2]
```diff
@@ -848,6 +945,7 @@ def hostname(value):
         dest='role',
         help="Set role"
     )
+    parser.add_argument('--is-qa-demo', action='store_true', help="Flagged as qa demo")
```
Comment (hitz) [inline src/encoded/commands/deploy.py:948]: does this actually do anything but tag the instance? (apparently true!)

**Review by hitz — APPROVED** [review]

**Pull request merged**